src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -11,19 +9,11 @@ import mesosphere.marathon.core.instance.Instance
 import mesosphere.marathon.state.{AppDefinition, PathId, Timestamp}
 import mesosphere.marathon.core.deployment.{DeploymentPlan, DeploymentStep}
 import org.apache.mesos.{Protos => Mesos}
-import play.api.libs.json.Json
-
 import scala.collection.immutable.Seq
 
 sealed trait MarathonEvent {
   val eventType: String
   val timestamp: String
-
-  @JsonIgnore
-  lazy val fullJsonString: String = Json.stringify(eventToJson(this, false))
-
-  @JsonIgnore
-  lazy val lightJsonString: String = Json.stringify(eventToJson(this, true))
```

Comment:
I think we should keep this for caching purposes. We send events to several consumers and do not want to serialize it each time.

Reply:
I think it's still there, just renamed, so we should be fine

Reply:
Actually I removed this in my initial commit(s). Brought it back after @jeschkies pointed out that we need it. Git diff wasn't smart enough to mark this comment as "Outdated" even after I pushed my commits.